VideoWeb/VideoWeb/ClientApp/src/app/shared/force-play-video.directive.ts

```diff
@@ -36,8 +38,12 @@ export class ForcePlayVideoDirective implements OnInit, OnDestroy {
         this.unsubscribeFromTouchStartCallback = this.renderer.listen('window', 'touchstart', this.onMouseDownOrTouchStart.bind(this));
 
         this.videoElement.oncanplay = event => {
-            this.logger.info(`${this.loggerPrefix} - videoElement.oncanplay - playing video`);
-            this.videoElement.play();
+            this.logger.info(`${this.loggerPrefix} - videoElement.oncanplay - event triggered`);
+
+            if (!this.destroyed) {
```

Comment:
I did wonder if this would still be available after ngOnDestroy was called. We have tested and it works.